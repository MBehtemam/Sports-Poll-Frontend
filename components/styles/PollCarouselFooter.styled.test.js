import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import { mount, shallow } from "enzyme";
import "jest-styled-components";
import PollCarouselFooter from "./PollCarouselFooter.styled";

describe("Test suits for PollCarouselFooter component", () => {
  it("should match with snapshot", () => {
    const tree = renderer.create(<PollCarouselFooter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("should has correct rule for childeren", () => {
    const tree = mount(
      <PollCarouselFooter>
        <div>First</div>
        <div>Second</div>
      </PollCarouselFooter>
    );
    expect(tree.find("div")).toHaveLength(2);
  });
});
