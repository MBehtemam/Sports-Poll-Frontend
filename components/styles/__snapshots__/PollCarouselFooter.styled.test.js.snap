// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for PollCarouselFooter component should match with snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 div {
  margin-bottom: 30px;
}

<footer
  className="c0"
/>
`;
